texastribune/styleguides
# Adds Grunt to style guides

#### What's the plan?

Add Grunt to style guides since Bower from other PR wasn't working.
#### What's this PR do?

Adds in Grunt, festival Sass, TribTalk Sass, and Gemfile. Not sure why the commit from 7/29 is being included in this PR, but that h2 should be part of site footer typography section for TribTalk, so just leaving it.
#### How should this be manually tested?
- Run `npm install && bundle install`
- Then run `grunt`, `grunt dev`, `grunt build`, and `grunt sass` to check that tasks are running as desired.
